### DISCUSSION TITLE
dio0 interrupt happen when no signal received

### DISCUSSION THREAD

**Discussion by aiotfactory** [Lora-net/LoRaMac-node#1183] [Q&A]
hello, 

We used a mcu to control sx1276, and it can tx and rx smoothly. To save power, the mcu will enter deep sleep and sx1276's dio pin can wake up the mcu, and before mcu sleeps, we set sx1276 into continuous rx mode, but the mcu is waken up after around 1 hour though there is no any lora signal received.

Please kindly help.
thanks.
Peter

**Comment by mluis1** [comment]
We never observed similar issue with this project supported platforms.

In case you are not using one of the currently supported platforms I would recommend that you review your platform port files.
I guess that you need to verify how the MCU GPIOs are setup before entering and after exiting the sleep mode.

In the future it would be nice if you could post this kind of questions on the project [Discussions](https://github.com/Lora-net/LoRaMac-node/discussions) tab. It is a better place to engage discussions and then we can agree if it is an issue or not.
If you don't mind I move this issue to the [Discussions](https://github.com/Lora-net/LoRaMac-node/discussions) tab.
